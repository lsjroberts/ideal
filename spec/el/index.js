import { el, text } from 'ideal-ui';

export default el([], text`hello`);
