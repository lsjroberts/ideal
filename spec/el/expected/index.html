<!DOCTYPE html>
<html>
  <head>
    <title>text</title>
    <style type="text/css">
      html,
      body {
        margin: 0;
        padding: 0;
        height: 100%;
      }

      .s {
        position: relative;
        border: none;
        flex-shrink: 0;
        display: flex;
        flex-direction: row;
        flex-basis: auto;
        resize: none;
        box-sizing: border-box;
        margin: 0;
        padding: 0;
        border-width: 0;
        border-style: solid;
        font-size: inherit;
        color: inherit;
        font-family: inherit;
        line-height: 1;
        font-weight: inherit;
        text-decoration: none;
        font-style: inherit;
      }

      .s.e {
        display: flex;
        flex-direction: column;
        white-space: pre;
      }
    </style>
  </head>
  <body>
    <div class="s e">hello</div>
  </body>
</html>
